mne_bids/path.py

```diff
@@ -255,17 +254,18 @@ def update(self, **entities):
         If one creates a bids basename using
         :func:`mne_bids.make_bids_basename`:
 
-        >>> bids_basename = make_bids_basename(subject='test', session='two', 
-        >>>                                    task='mytask', kind='data', extension='.csv')
```

Comment:
This was all good! You just would've needed to remove the `>>>` in the second line :)